Tests/Testo/Invoice/Inv/InvServiceTest.php

```
@@ -0,0 +1,672 @@
+            'date_tax_point' => '2026-05-03',
+        ];
+
+        $repo = m::mock(InvRepository::class);
+        /** @var \Mockery\Expectation $e */
+        $e = $repo->shouldReceive('save');
+        $e->once()->with($model);
+
+        $service = $this->makeService($repo);
+        $service->saveInvFromRecurring($user, $model, $details, $s);
+
+        Assert::same(1, $model->reqStatusId());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Testo\Assert' in method 'saveInvFromRecurringOnNewInvoiceResetsCoreFieldsToFreshDefaultsAndSaves'. (undefined)

(StaticAccess)